script/intentfest/conversation.py

```diff
@@ -0,0 +1,145 @@
+
+
+class InstructionsI18n:
+    def __init__(self, languageKey="en") -> None:
+        self.fallbackLanguage = "en"
+
+        # Feel free to add texts in your language ;)
+        self.instructionsI18n = {
+            "en": {
+                "HELLO_TEXT": 'Write here your query, or type \'a\' (or "arrow up" on keyboard), to repeat the last one question. Type in \'r\' to reload intents (after changing "sentences" or "responses").',
+                "DEBUG": "Debug",
+                "DEBUG_ON": "ON",
+                "DEBUG_OFF": "OFF",
+                "NEW_INTENTS_RELOADED": "New intents reloaded!",
+                "ERROR_OCCURRED": "Oops! There's been an error:",
+                "NO_AVAILABLE_RESPONSE": "(Sorry, there is no response for this)",
+            },
+            "pl": {
+                "HELLO_TEXT": "Napisz o co checesz się zapytać, ewentualnie napisz 'a' (lub sztrzałka w górę na klawiaturze), aby powtórzyć ostatnie zapyatnie. Możesz też napisać 'r', aby przeładować pliki językowe (np. po zmianie \"sentences\", lub \"responses\").",
+                "DEBUG": "Debug",
+                "DEBUG_ON": "ON",
+                "DEBUG_OFF": "OFF",
+                "NEW_INTENTS_RELOADED": "Przeładowano pliki językowe!",
+                "ERROR_OCCURRED": "Whoops! Wystrąpił bląd:",
+                "NO_AVAILABLE_RESPONSE": "(niestety, brak odpowidzi)",
+            },
+        }
+
+        self.language = None
+        if languageKey not in self.instructionsI18n:
+            self.language = "en"
+        else:
+            self.language = languageKey
+
+    def get(self, key):
+        languageSet = self.instructionsI18n[self.language]
+        if key not in languageSet:
+            fallbackLanguageSet = self.instructionsI18n[self.fallbackLanguage]
+            if key not in fallbackLanguageSet:
+                raise Exception(
+                    f'Can\'t find the key "{key}" in language set "{self.language}" (and in default language set "{self.fallbackLanguage}" also).'
+                )
+            return fallbackLanguageSet[key]
+        return languageSet[key]
+
+
+class Conversation:
+    def __init__(self, language, debug=False) -> None:
+        self.language = language
+        self.localDebug = False if debug is None or debug is False else bool(int(debug))
+
+        self.sentenceParser = SentenceParser(self.language)
+        self.instructionsI18n = InstructionsI18n(self.language)
+
+        self.lastSentence = ""
+
+    def askUser(self):
+        promptSign = "<<<"
+        userSentence = input(f"{promptSign} ")
+
+        # Toggle debug
+        if userSentence.lower() == "d":
+            self.localDebug = not self.localDebug
+            print(
+                f"{self.instructionsI18n.get('DEBUG')}: {self.instructionsI18n.get('DEBUG_ON') if self.localDebug else self.instructionsI18n.get('DEBUG_OFF')}"
+            )
+
+        # Do intents reload
+        elif userSentence.lower() == "r":
```

Comment:
Instead of supporting reload, we should add [`watchdog`](https://pypi.org/project/watchdog/) to automatically reload + reparse last sentence.

Reply:
Wow! Good point! I use [nodemon](https://www.npmjs.com/package/nodemon) all the time, but I hadn't thought of that!

Reply:
The script now listens for file changes! Thanks!